### ISSUE TITLE
"select" comand

### ISSUE THREAD

**Issue by vjrj** [smblott-github/chromix-too#12]
Would be nice to have a select command that doesn't focus the window, it only select the tab (useful if you have two screens and you don't want to lost the focus on your terminal or editor). Something like:

```diff
diff --git a/client.coffee b/client.coffee
index ee5160f..16bc7f1 100644
--- a/client.coffee
+++ b/client.coffee
@@ -87,6 +87,11 @@ switch commandName
         chromix "chrome.tabs.update", {}, tab.id, {selected: true}
         focusWindow tab.windowId
 
+  when "select"
+    getMatchingTabs commandArgs, (tabs) ->
+       for tab in tabs
+         chromix "chrome.tabs.update", {}, tab.id, {selected: true}
+
   when "reload"
     getMatchingTabs commandArgs, (tabs) ->
       chromix "chrome.tabs.reload", {}, tab.id, {} for tab in tabs
```

For some reason, I cannot send a PR because it only works when I manually patch the npm package.

**Comment by smblott-github** [comment]
Is that not doing pretty much the same thing as the `focus` and `activate` commands?

**Comment by vjrj** [comment]
I don't think so, because it just select the tab. If you have chrome in other screen, you can only see the tab without losing the focus in your editor.
In my case, I'm typing in my terminal selecting what to do with each tab (close, skip, or archive in a bookmark app).

One more thing: thanks indeed for your chromix ;-)

**Comment by smblott-github** [comment]
Fixed and published to npm (version 0.0.10).
